TeamCharles/customer-service
# 19 create incident user story

## Status
**READY**

## Testing

[ ] I have added passing tests for all new functionality.

[X] I have confirmed that all test prior to this pull request still PASS.

## Documentation

[ ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[X ] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

This partially resolves issue number 19. I think that this issue needs to be broken down into smaller issues. So, this one partially resolves the issue at hand.

## Resolves Issue Number

#19 

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

You will need to add these methods into main on program

```
            LoginAction.ReadInput();
            CreateIncidentAction.ReadInput();
```

```sh
git pull --prune
git checkout 19-create-incident-user-story
dotnet run
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Run the program and add 2 different incidents.
2. You can see that this program works, because the numbers increment properly 

## Impacted Files in Application
List general components of the application that this PR will affect:

* actions/CreateIncidentAction.cs
